indico/modules/admin/__init__.py

```diff
@@ -34,7 +34,8 @@ def _sidemenu_sections(sender, **kwargs):
 def _sidemenu_items(sender, **kwargs):
     yield SideMenuItem('general', _('General Settings'), url_for('admin.adminList'), 100, icon='settings')
     yield SideMenuItem('storage', _('Disk Storage'), url_for('admin.adminSystem'), 70, icon='stack')
-    yield SideMenuItem('ip_domains', _('IP Domains'), url_for('admin.domainList'), section='security')
+    yield SideMenuItem('ip_domains_old', _('IP Domains (old)'), url_for('admin.domainList'), section='security')
+    yield SideMenuItem('ip_domains', _('IP Networks'), url_for('networks.manage'), section='security')
```

Comment:
`networks` or `ip_networks` and move it to the networks module (just connect the same signal in there and return/yield just that specific item)